src/index.jsx

```diff
@@ -30,8 +30,12 @@ class ReactSwitch extends Component {
     this.$onTouchEnd = this.$onTouchEnd.bind(this);
     this.$onTouchCancel = this.$onTouchCancel.bind(this);
 
-    this.$onClick = this.$onClick.bind(this);
-    this.$onKeyDown = this.$onKeyDown.bind(this);
+    this.$onInputChange = this.$onInputChange.bind(this);
+
+    this.setHasOutline = this.setHasOutline.bind(this);
```

Comment:
So the reason that all the methods are prefixed with $-sign is to mark them for uglify-js so they can be mangled to one character in production mode (to make bundle size as small as possible). Some would say that's overkill but let's be consistent in the repo and prefix all methods with $.

Reply:
Ok, did not realize the reasoning for the prefixing. Consider it done